addons/event-audit/common/src/main/java/org/commonjava/indy/event/audit/EventConstants.java

```diff
@@ -0,0 +1,24 @@
+package org.commonjava.indy.event.audit;
+
+public class EventConstants
```

Comment:
Would it make sense to turn any of these into first-class fields in the FileEvent type, rather than entries in a map? I'm not sure if any of them are universal (or nullable) without making things confusing...I guess by making them entries in a map, we're effectively saying they're all nullable.